### DISCUSSION TITLE
UI and feature of the search area

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#22] [General]
☐ recherche
    ☐ 1 champ "tout" (comme l'actuel)
    ☐ spinner
    ☐ plus tard et optionnel : si possible, les champs de recherche sont dans un fichier geojson
    ☐ chercher dans geojson ceramique, eventuellement dans secteur
    ☐ UI : un champs recherche en surimpression, plus caret qui ouvre le multicritere. (Probleme : si on ferme, on ne voit plus qu'il y a des criteres)

**Comment by AlZorba** [comment]
IDig utilise un fichier json qui défini les champs : leur nom, leurs "label" dans chaque langue, leur type (valeur unique, multiple, numéro etc) et d'autre critères possible comme "searchable".
Un json similaire pourrait définir les champs de recherche dont le composant se servirait pour afficher le module recherche.
